core/core.go

```diff
@@ -63,6 +64,9 @@ var (
 	blockMethods = []string{
 		"/threads.pb.API/ListDBs",
 	}
+
+	// wsPingInterval controls the WebSocket keepalive pinging interval. Must be >= 1s.
+	wsPingInterval = time.Second * 5
```

Comment:
If you make this exported, users can tweak it... an easy win

Reply:
🤦 of course! good call